OCR/frontend/src/pages/ReviewTemplate.tsx

```diff
@@ -202,25 +217,34 @@ const ReviewTemplate: React.FC = () => {
         </div>
         <div className="width-50">
           <div
-            style={{
```

Comment:
Removing this is what's causing the playwright test to break, I'd suggest removing the test `Document image scrollable` as it seems that is no longer a feature we have.